content/en/docs/tutorials/stateful-application/elasticsearch.md

```diff
@@ -0,0 +1,380 @@
+---
+title: "Building a Search Engine on Kubernetes using Elasticsearch"
+content_type: tutorial
+weight: 60
+---
+
+<!-- overview -->
+
+{{< warning >}}
+This deployment is not suitable for production use cases, you need to benchmark or size your cluster according to your use case.
+{{< /warning >}}
+
+{{< note >}}
+The files provided in this tutorial are using GA Deployment APIs and are specific to kubernetes version 1.16 and 1.20. You can also try it on newer Kubernetes version but some APIs might throw deprecated warnings. 
+{{< /note >}}
```

Comment:
Hi. These look like prerequisites - I'd mention them in the prerequisites section.